workflows/sarek.nf

```diff
@@ -1151,9 +1063,9 @@ workflow SAREK {
         ch_multiqc_files = ch_multiqc_files.mix(ch_version_yaml)
         ch_multiqc_files = ch_multiqc_files.mix(ch_workflow_summary.collectFile(name: 'workflow_summary_mqc.yaml'))
         ch_multiqc_files = ch_multiqc_files.mix(ch_methods_description.collectFile(name: 'methods_description_mqc.yaml'))
-        ch_multiqc_files = ch_multiqc_files.mix(ch_reports.collect().ifEmpty([]))
+        ch_multiqc_files = ch_multiqc_files.mix(reports.collect().ifEmpty([]))
 
-        ch_multiqc_configs = Channel.from(ch_multiqc_config).mix(ch_multiqc_custom_config).ifEmpty([])
+        ch_multiqc_configs = ch_multiqc_config.mix(ch_multiqc_custom_config).ifEmpty([])
```

Comment:
In 906 this changes to Channel.of, right? Seems like it might all together not be needed anymore?

Reply:
yes, I can remove this part